string.go

```diff
@@ -1,13 +1,14 @@
 package lo
 
 import (
+	"math"
+	"math/rand"
```

Comment:
You might use math/rand/v2 now

https://go.dev/blog/randv2

As you did a benchmark, you could also compare the speed.

Reply:
thank you
```go
func BenchmarkApproach1(b *testing.B) {
	for i := 0; i < b.N; i++ {
		_ = RandomString(1<<10, lo.AlphanumericCharset)
	}
}

func BenchmarkApproach2(b *testing.B) {
	for i := 0; i < b.N; i++ {
		_ = lo.RandomString(1<<10, lo.AlphanumericCharset)
	}
}

func BenchmarkApproach3(b *testing.B) {
	for i := 0; i < b.N; i++ {
		_ = RandomStringV2(1<<10, lo.AlphanumericCharset)
	}
}
```
```go
$ go test -benchmem -benchtime=5s -bench .
goos: windows
goarch: amd64
pkg: mytttttt
cpu: Intel(R) Core(TM) i5-10400 CPU @ 2.90GHz
BenchmarkApproach1-12            1000000              5509 ns/op            1024 B/op          1 allocs/op
BenchmarkApproach2-12             291853             20599 ns/op            5248 B/op          2 allocs/op
BenchmarkApproach3-12            1000000              5458 ns/op            1024 B/op          1 allocs/op
PASS
ok      mytttttt        17.782s
```

Reply:
Quite comparable then. Unless I'm reading wrong the benchmark.

Maybe using the v2 even if it's a bit slower

Reply:
thanks, I have upgraded math/rand to math/rand/v2